### DISCUSSION TITLE
Curious Why there is a pin going nowhere Between the White and Green wires coming from the DF Robot Rotary Encoder

### DISCUSSION THREAD

**Discussion by daveclifford** [DCC-EX/EX-T3-WiFi#51] [General]
![image](https://user-images.githubusercontent.com/1976894/216864948-b4b068c5-6a12-4861-bd07-6ccf04ab0249.png)

Was that so you put the other pins in the correct sequence in the FireBeetle2 ????????  

Dave

**Comment by daveclifford** [comment]
Also another unused pin between the red and black wires on the lower part of the FireBeetle2

**Reply by alex-code** [reply]
The pin between the White and Green wires (just the colours I used, no significance) is A2, this is already wired as a voltage divider on the FireBeetle board so couldn't be used.

The pin between the Red and Black wires isn't connected to anything (labelled as NC).

Using a full row of angled headers just gives more strength as a single one is more likely to twist.

**Reply by daveclifford** [reply]
Nice! Very nice!! Smart! Very Smart!!!!!!!!!!!

**Comment by daveclifford** [comment]
I guess that, with my question on the unused connectors, you are now aware that I am near complete and in the final wiring stage.

I will take several pictures and put them in the Show and Tell section (like I'm back in Rutherford, NJ - Union Elementary School - 3rd Grade - circa 1953-54)

Dave
